### PR TITLE
Update credits for 5.6

### LINKED ISSUE #260: Update credits page
current in-game credits: https://github.com/minetest/minetest/blob/master/builtin/mainmenu/tab_about.lua#L20

perhaps remove the "Contributors" sections so we don't have to update it as often

also I don't know where this page is even linked from, it should be made more prominent _or_ removed entirely.

### PR DESCRIPTION
**Pull request by rubenwardy**
This adds credits.yml, using [] syntax so that the lines can just be copied from tab_about.lua with a couple of modifications

Fixes #260

### CODE REVIEW

File: _data/credits.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+active_core_devs: [
+  "Perttu Ahola (celeron55) &#x3C;<a href='mailto:&#x63;&#x65;&#x6C;&#x65;&#x72;&#x6F;&#x6E;&#x35;&#x35;&#x40;&#x67;&#x6D;&#x61;&#x69;&#x6C;&#x2E;&#x63;&#x6F;&#x6D;' title='&#x63;&#x65;&#x6C;&#x65;&#x72;&#x6F;&#x6E;&#x35;&#x35;&#x40;&#x67;&#x6D;&#x61;&#x69;&#x6C;&#x2E;&#x63;&#x6F;&#x6D;'>&#x63;&#x65;&#x6C;&#x65;&#x72;&#x6F;&#x6E;&#x35;&#x35;&#x40;&#x67;&#x6D;&#x61;&#x69;&#x6C;&#x2E;&#x63;&#x6F;&#x6D;</a>&#x3E;",
```
Comment (appgurueu) [inline _data/credits.yml:2]: Why the obfuscation? Anything scraping the website (and using a correct HTML parser) will obtain these addresses just fine. I also don't like embedding HTML here; why not use a clean YAML instead and generate the proper HTML in the template?

Comment (rubenwardy) [reply _data/credits.yml:2]: I haven't changed this, it was encoded like this before

Comment (sfan5) [reply _data/credits.yml:2]: I bet you could script this but also it doesn't matter

**Review by sfan5 — APPROVED** [review]

File: _data/credits.yml (hunk 1 above)
Comment (appgurueu) [reply _data/credits.yml]: Done

**Review by sfan5 — APPROVED** [review]

**Pull request merged**